src/Middleware/HttpLogging/src/FileLoggerOptions.cs

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using Microsoft.Extensions.Hosting;
+
+namespace Microsoft.AspNetCore.HttpLogging
+{
+    /// <summary>
+    /// Options for the <see cref="W3CLogger"/>.
+    /// </summary>
+    public class FileLoggerOptions
```

Comment:
name might be too generic, could wait for API review to bike-shed

Reply:
They're joined now